.travis.yml

```diff
@@ -27,7 +27,7 @@ before_install:
   - pip install -r requirements/test.txt
   - pip install codecov
 install:
-  - pip install .
+  - pip install -e .
```

Comment:
🤔  Except now it will not test for missing manifest files, right?

Reply:
I guess so... :sweat:

Reply:
I don't see a way around this though. I've recently not noticed that coverage.py doesn't work on installed packages. So we either get coverage or we test for installation of files.

I swear this worked not that long ago.

Reply:
I looked on travis and for python 2.7, coverage broke around the time you moved the `tests/` directory (from `cnxdb/tests`), which is quite a long time ago.